sdk/python/lib/pulumi/runtime/invoke.py

```diff
@@ -66,10 +66,20 @@ def _sync_await(awaitable: Awaitable[Any]) -> Any:
 
     # Pump the event loop until the future is complete. This is the kernel of BaseEventLoop.run_forever, and may not
     # work with alternative event loop implementations.
+    #
+    # In order to make this reentrant with repsect to _run_once, we keep track of the number of event handles on the
```

Comment:
`r` `e` `p` `s` `e` `c` `t`, find out what it means to me.

Reply:
`s` `p` `e` `c` `t` !